src/map.cpp

```diff
@@ -4579,12 +4579,12 @@ void map::make_active( item_location &loc )
 void map::apply_in_fridge( item &it, int temp )
 {
     unsigned int diff_freeze = abs(temp - FREEZING_TEMPERATURE);
-    diff_freeze = diff_freeze < 1 ? 1 : diff_freeze;
-    diff_freeze = diff_freeze > 5 ? 5 : diff_freeze;
+    diff_freeze = std::max( 1, diff_freeze );
```

Comment:
You can't put int and unsigned int together.

![image](https://user-images.githubusercontent.com/16213433/44286300-4b1d0f80-a271-11e8-9bfa-e36a2cb9d8b0.png)

Reply:
just noticed that in a compiler, sorry